app/src/router/index.ts

```diff
@@ -16,6 +16,7 @@ const routes: Array<RouteRecordRaw> = [
   { path: '/dgrants', name: 'dgrants', component: () => import('../views/GrantRegistryList.vue') },
   { path: '/review', name: 'review', component: () => import('../views/GrantReview.vue') },
   { path: '/dgrants/new', name: 'dgrants-new', component: () => import('../views/GrantRegistryNewGrant.vue') },
+  { path: '/dgrants/mygrants', name: 'dgrants-my-grants', component: () => import('../views/MyGrants.vue') },
```

Comment:
Nit on path slug, this would be my personal preference 
```suggestion
  { path: '/dgrants/my-grants', name: 'dgrants-my-grants', component: () => import('../views/MyGrants.vue') },
```